src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxNode.cs

```diff
@@ -904,17 +717,17 @@ protected override SyntaxTree SyntaxTreeCore
 
         protected override SyntaxToken FindTokenCore(int position, bool findInsideTrivia)
```

Comment:
@agocke Can i remove this method?  Is it a breaking change to get rid of an override?  I wasn't sure, so i left it in.